build.gradle.kts

```diff
@@ -44,3 +44,6 @@ kover {
         }
     }
 }
+tasks.dokkaHtmlMultiModule {
```

Comment:
Could you please move this config setup to [kotlinx-io-dokka.gradle.kts](https://github.com/Kotlin/kotlinx-io/blob/master/build-logic/src/main/kotlin/kotlinx/io/conventions/kotlinx-io-dokka.gradle.kts) too?
I guess, `dokkaHtmlMultiModule` would need to be replaced with `withType(DokkaMultiModuleTask::class.java)`.